persist/file-adapter/adapter_filtered.go

```diff
@@ -34,8 +34,7 @@ type FilteredAdapter struct {
 // Filter defines the filtering rules for a FilteredAdapter's policy. Empty values
 // are ignored, but all others must match the filter.
 type Filter struct {
-	P []string
-	G []string
+	PG map[string][][]string
```

Comment:
This is a breaking change. At least we should use "feature:" instead of "fix:"

Reply:
For: https://github.com/casbin/casbin/pull/826#discussion_r661350076, we can use fix: